package.json

```diff
@@ -268,7 +268,7 @@
     "@metamask/browser-passworder": "^4.3.0",
     "@metamask/contract-metadata": "^2.5.0",
     "@metamask/controller-utils": "^11.4.0",
-    "@metamask/design-tokens": "^5.0.0",
+    "@metamask/design-tokens": "^5.1.0",
```

Comment:
Upgrading design tokens from [V3 release](https://github.com/MetaMask/metamask-design-system/pull/476) of design system monrepo